### PR TITLE
modify ctime and mtime of md files after copying

### PR DESCRIPTION
**Pull request by hkievet**
This PR augments evernote2md so that the markdown files it generates have a ctime and mtime that reflect the evernote xml data.

On Windows, only the mdate will be changed using the stdlib's `os.Chtime` (did not personally test).

### CODE REVIEW

**Review by wormi4ok — COMMENTED** [review]
Thank you very much for your contribution, @hkievet!

Event though I left a few comments, you did a good job. I'm happy to accept the PR as is and make a few tweaks to intergate the changes in the codebase later on, since there is no written code style or contribution guidelines for the project.

I also researched this topic a bit and found [SetFileTime](https://golang.org/src/syscall/zsyscall_windows.go#L874) syscall, that could solve the problem for Windows.

File: file/file.go [hunk 1]
```diff
@@ -37,6 +43,33 @@ func Save(dir, name string, content io.Reader) error {
...
+func ChangeFileTimes(dir, name string, ctime, mtime time.Time) {
+	var err error
+	filePathToModify := filepath.FromSlash((dir + "/" + name))
+	_, err = os.Stat(filePathToModify)
+	if os.IsNotExist(err) {
+		// file doesn't exist
+		log.Printf("Tried to change file creation times, file not found %s", filePathToModify)
+		return
+	}
+	_, err = exec.LookPath("touch")
```
[the remark this answers is not in the document]
Comment (hkievet) [reply file/file.go:58]: Go makes this really easy!

**Comment by hkievet** [comment]
@wormi4ok Thanks for all the feedback!  I made the requested changes and updated the PR

**Review by wormi4ok — APPROVED** [review]
Amazing. Happy to merge it! I'll spend some time testing this logic and aim to publish a new release by the end of the week.

**Pull request merged**